### ISSUE TITLE
Image is not displaying in the chat window 

### ISSUE THREAD

**Issue by comradesurendra** [jaewonhimnae/react-chat#13]
when trying to send the image or video in the chat it is not displaying in the chat window instead it is showing the file name.

**Comment by josephhk** [comment]
I'm guessing you are on a windows computer. His substring method for determining the file type doesn't let the image show on windows but it can be easily fixed by changing the substring and/or statement.

**Comment by comradesurendra** [comment]
Yes, i am using windows. But still not able to resolve it. i tried what you told but still, it is not showing the image and video in the chat window.

**Comment by idsus** [comment]
I have same error
